include/uapi/sound/sof/abi.h

```diff
@@ -27,7 +27,7 @@
 /* SOF ABI version major, minor and patch numbers */
 #define SOF_ABI_MAJOR 3
 #define SOF_ABI_MINOR 17
-#define SOF_ABI_PATCH 0
+#define SOF_ABI_PATCH 1
```

Comment:
I think you can drop this change to ABI_PATCH. The FW version this affects is 3.18.1, so it will get a bit complicated to boost the version in this patch for kernel, before other changes that add 3.18 features to kernel, get merged.